apache/couchdb-fauxton
# Fix: accents are ignored in docs search box

## Overview

The documents search box doesn't work for IDs that contain characters with accents.
For instance, when typing `á` in the search box, it won't list the docs that start with that character.

## Testing recommendations

 - Create a new doc using `{"_id": "á"}`
 - Type `á` in the search box. It should list the doc you created


## Checklist
